src/storage/store/rel_table.cpp

```diff
@@ -355,54 +365,26 @@ void RelTable::prepareCommitForDirection(RelDirection relDirection) {
                 }
                 auto adjLists = getAdjLists(relDirection, boundNodeTableID);
                 if (listsUpdatesForNodeOffset->isNewlyAddedNode) {
-                    auto inMemAdjLists = adjLists->createInMemListWithDataFromUpdateStoreOnly(
-                        nodeOffset, listsUpdatesForNodeOffset->insertedRelsTupleIdxInFT);
-                    listsUpdateIterators[0]->updateList(nodeOffset, *inMemAdjLists);
-                    auto numPropertyLists = getNumPropertyLists(relDirection, boundNodeTableID);
-                    for (auto i = 0u; i < numPropertyLists; i++) {
-                        auto inMemPropLists =
-                            getPropertyLists(relDirection, boundNodeTableID, i)
-                                ->createInMemListWithDataFromUpdateStoreOnly(nodeOffset,
-                                    listsUpdatesForNodeOffset->insertedRelsTupleIdxInFT);
-                        listsUpdateIterators[i + 1]->updateList(nodeOffset, *inMemPropLists);
-                    }
+                    prepareCommitForListWithUpdateStoreDataOnly(adjLists, nodeOffset,
+                        listsUpdatesForNodeOffset.get(), relDirection, listsUpdateIterators,
+                        boundNodeTableID, updateList);
```

Comment:
I think it's good to put the word "op" or "fn" to function arguments to clarify that they are functions. So either updateListOp or opUpdateList or fnUpdateList etc. Also for appendInMemListToLargeList.

Reply:
Added op to `appendInMemListToLargeList` and `updateList`